<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookDetailsDialog</class>
 <widget class="QDialog" name="BookDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Подробнее о книге</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="headerlayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="header">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Название книги</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnAddFav">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="toolTip">
             <string>1 звезда</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="bodyContainer" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QFrame" name="leftPane">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="lblCover">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>280</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>260</width>
                <height>360</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="text">
               <string>pixmap</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblAuthor">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text">
               <string>Автор: …</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblTags">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Теги: Бизнес, Менеджмент, …</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QFrame" name="rightPane">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lblSubtitle">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Описание:</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="teDescription">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>220</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QWidget" name="ratingContainer" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="ratingLayout">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="lblRatingTitle">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Ваша оценка:</string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QWidget" name="starsContainer" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>0</height>
                    </size>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <layout class="QHBoxLayout" name="starsLayout">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QToolButton" name="btnStar1">
                        <property name="toolTip">
                         <string>1 звезда</string>
                        </property>
                        <property name="text">
                         <string>...</string>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>54</width>
                          <height>54</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoRaise">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QToolButton" name="btnStar2">
                        <property name="toolTip">
                         <string>2 звезды</string>
                        </property>
                        <property name="text">
                         <string>...</string>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>54</width>
                          <height>54</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoRaise">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QToolButton" name="btnStar3">
                        <property name="toolTip">
                         <string>3 звезды</string>
                        </property>
                        <property name="text">
                         <string>...</string>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>54</width>
                          <height>54</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoRaise">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QToolButton" name="btnStar4">
                        <property name="toolTip">
                         <string>4 звезды</string>
                        </property>
                        <property name="text">
                         <string>...</string>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>54</width>
                          <height>54</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoRaise">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QToolButton" name="btnStar5">
                        <property name="toolTip">
                         <string>5 звезд</string>
                        </property>
                        <property name="text">
                         <string>...</string>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>54</width>
                          <height>54</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoRaise">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
